### PR TITLE
system tables: update diagnose system table doc

### PR DESCRIPTION
**Pull request by toutdesuite**
### What is changed, added or deleted? (Required)

update diagnose system table doc

### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3154
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
Please help review this PR. @TomShawn

File: system-tables/system-table-metrics-schema.md [hunk 1]
```diff
@@ -7,11 +7,11 @@ aliases: ['/docs/dev/reference/system-databases/metrics-schema/']
...
 
 ## Overview
 
-The following example uses the `tidb_query_duration` table to introduce the usage and working principles of the monitoring table. The working principles of other monitoring tables are similar.
+To illustrate how to use the monitoring table and how it works, take the `tidb_query_duration` monitoring table in `metrics_schema` as an example. The principles of other monitoring tables are similar to `tidb_query_duration`.
```
Comment (TomShawn) [inline system-tables/system-table-metrics-schema.md:14]:
```suggestion
Taking the `tidb_query_duration` monitoring table in `metrics_schema` as an example, this section illustrates how to use this monitoring table and how it works. The working principles of other monitoring tables are similar to `tidb_query_duration`.
```

File: system-tables/system-table-metrics-schema.md [hunk 2]
```diff
@@ -31,8 +31,8 @@ select * from information_schema.metrics_tables where table_name='tidb_query_dur
 
 Field description:
 
-* `TABLE_NAME`: Corresponds to the table name in the metrics schema. In this example, the table name is `tidb_query_duration`.
-* `PROMQL`: The working principle of the monitoring table is to map SQL statements to `PromQL` and convert Prometheus results into SQL query results. This field is the expression template of `PromQL`. When getting the data of the monitoring table, the query conditions are used to rewrite the variables in this template to generate the final query expression.
+* `TABLE_NAME`: Corresponds to the table name in `metrics_schema` . In this example, the table name is `tidb_query_duration`.
+* `PROMQL`: The working principle of the monitoring table is to first map SQL statements to `PromQL`, then request data from Prometheus, and convert Prometheus results into SQL query results. This field is the expression template of `PromQL`. When you query the data of the monitoring table, the query conditions are used to rewrite the variables in this template to generate the final query expression.
```
Comment (TomShawn) [inline system-tables/system-table-metrics-schema.md:35]:
```suggestion
* `PROMQL`: The working principle of the monitoring table is to first map SQL statements to `PromQL`, then to request data from Prometheus, and to convert Prometheus results into SQL query results. This field is the expression template of `PromQL`. When you query the data of the monitoring table, the query conditions are used to rewrite the variables in this template to generate the final query expression.
```

File: system-tables/system-table-sql-diagnosis.md [hunk 3]
```diff
@@ -48,11 +48,11 @@ To dynamically observe and compare cluster conditions in different time periods,
 Because the TiDB cluster has many monitoring metrics, TiDB provides the following monitoring summary tables in v4.0:
 
 + The monitoring summary table [`information_schema.metrics_summary`](/system-tables/system-table-metrics-summary.md) summarizes all monitoring data to for you to check each monitoring metric with higher efficiency.
-+ The monitoring summary table [`information_schema.metrics_summary_by_label`](/system-tables/system-table-metrics-summary.md)) also summarizes all monitoring data, but this table performs differentiated statistics according to different labels.
++ [`information_schema.metrics_summary_by_label`](/system-tables/system-table-metrics-summary.md)) also summarizes all monitoring data. Particularly, this table aggregates statistics using different labels of each monitoring item.
```
Comment (TomShawn) [inline system-tables/system-table-sql-diagnosis.md:51]: Looks like our works overlap... https://github.com/pingcap/docs/pull/2772
```suggestion
+ [`information_schema.metrics_summary_by_label`](/system-tables/system-table-metrics-summary.md)) also summarizes all monitoring data. Particularly, this table aggregates statistics using different labels of each monitoring metric.
```

**Pull request merged**